cli/src/pcluster/config/cluster_config.py

```diff
@@ -1829,6 +1870,43 @@ def queue_ami(self):
         else:
             return None
 
+    def get_managed_placement_group_keys(self) -> List[str]:
+        managed_placement_group_keys = []
+        if self.networking.placement_group.is_enabled_and_unassigned:
+            for resource in self.compute_resources:
+                if resource.networking.placement_group.should_be_managed:
+                    managed_placement_group_keys.append(f"{self.name}-{resource.name}")
+        else:
+            for resource in self.compute_resources:
+                if resource.networking.placement_group.is_enabled_and_unassigned:
+                    managed_placement_group_keys.append(f"{self.name}-{resource.name}")
+        return managed_placement_group_keys
+
+    def get_placement_group_key_for_compute_resource(
```

Comment:
I am wondering whether we should split this method to
`get_placement_group_name_for_compute_resource` and `is_placement_group_managed_for_compute_resource`?
Because this method has some repetitions of `"{self.name}-{compute_resource.name}"`, which are not really used in the consumers of this method.

Reply:
I wrote this function initially for the cluster_stack implementation, but I agree that splitting it might be cleaner for the other consumers to distinguish.

Reply:
will leave this for now